Downloads client/5.5.0 announcement (#296)
ome/www.openmicroscopy.org: _posts/2019-06-13-omero-5-5-0.md

---
layout: post
title: Release of OMERO 5.5.0
intro-blurb: The OME team is pleased to announce the release of OMERO 5.5.0.
---
Today we are releasing OMERO 5.5.0, a full production-ready release of OMERO.

This release also upgrades the version of Bio-Formats which OMERO uses to [6.1.0](https://docs.openmicroscopy.org/bio-formats/6.1.0/about/whats-new.html).

This version **does not** require a database upgrade.

> For more information about the aim of the 5.5 series and future plan, please read our [blog post](http://blog.openmicroscopy.org/)

Reviewer: ```suggestion
For more information about the aim of the 5.5 series and future plan, please read our [blog post](https://blog.openmicroscopy.org/future-plans/community/2019/06/03/omero-5-5/)
```

## User/client improvements

Additionally, this release improves the Web interface when OMERO is opened to the public
and contains some useful CLI improvements.

- web:
  - Introduce Advanced Search to allow and/or search options
  - Fix description in new Project, Dataset etc.
  - Allow user to create new Map Annotations for multiple selected objects at once
  - Fix date display
  - Remove / from 3rdparty in ome.viewportImage.js
  - Remove usage of deprecated calls
  - Remove apache config
  - Do not break display of Tag dialog when large font size is configured in browser
  - Disable refresh button while existing refresh in progress
  - Improve public user support
  - Add ability to customize incorrect login text

- cli:
  - Disable foreground indexing
  - Improve logging of error when importing data via cli command
  - Clearly indicate empty log files when running a diagnostic
  - Fix bug when running `config load` passing a directory instead of a file
  - Add option to delete keys from map annotations
  - Add error code discovery
  - Deprecate the CLI upload module and plugin

> The software is available at [archived downloads](https://downloads.openmicroscopy.org/omero/5.5.0). Clients are no longer available from that location.

Reviewer: worth mentioning why?

> The software is available at [archived downloads](https://downloads.openmicroscopy.org/omero/5.5.0). Clients are no longer available from that location.

Reviewer: ```suggestion
The software is available at [archived downloads](https://downloads.openmicroscopy.org/omero/5.5.0). Clients are no longer available from that location since they will be released more frequently than the server.
```

---
layout: post
title: Release of OMERO 5.5.0
intro-blurb: The OME team is pleased to announce the release of OMERO 5.5.0.
---
Today we are releasing OMERO 5.5.0, a full production-ready release of OMERO.

This release also upgrades the version of Bio-Formats which OMERO uses to [6.1.0](https://docs.openmicroscopy.org/bio-formats/6.1.0/about/whats-new.html).

This version **does not** require a database upgrade.

> For more information about the aim of the 5.5 series and future plan, please read our [blog post](https://blog.openmicroscopy.org/future-plans/community/2019/06/03/omero-5-5/)

Reviewer 2: ```suggestion
For more information about the aim of the 5.5 series and future plan, please read our [blog post](https://blog.openmicroscopy.org/future-plans/community/2019/06/03/omero-5-5/).
```
